projects/1.20-fabric/assets/xp-storage-trinkets/xp_storage_trinkets/lang/zh_cn.json

```
@@ -0,0 +1,13 @@
+{
+  "itemGroup.xp_storage_trinkets.trinkets": "XP Storage - Trinkets",
+
+  "item.xp_storage_trinkets.cristalis_lazuli": "晶化青金石",
+  "item.xp_storage_trinkets.xp_conduit": "经验手环",
+  "item.xp_storage_trinkets.xp_saver": "经验护符",
+
+  "text.autoconfig.xp_storage_trinkets.title": "XP Storage - Trinkets 配置",
+  "text.autoconfig.xp_storage_trinkets.option.xpConduitMaxDamage": "经验手环耐久度",
+  "text.autoconfig.xp_storage_trinkets.option.xpConduitTransfer": "经验手环转移损耗（%） ",
+  "text.autoconfig.xp_storage_trinkets.option.xpSaverMaxDamage": "经验护符耐久度",
+  "text.autoconfig.xp_storage_trinkets.option.xpSaverTransfer": "经验护符转移损耗（%）"

```

```suggestion
  "text.autoconfig.xp_storage_trinkets.option.xpSaverTransfer": "经验护符转移量（%）"

```
请问这个转移损耗是怎么来的？不应该是转移量吗？

---

我理解为了耐久度的损耗，进游戏里看了一下，确实是转移量